_posts/2024-07-22-developer-update.md

```diff
@@ -0,0 +1,115 @@
+---
+author: Jody Garnett
+layout: post
+title: GeoServer 2024 Q3 Developer Update
+date: 2024-07-22
+categories:
+- Behind The Scenes
+---
+
+This is a follow up to [2024 roadmap]({% post_url 2024-01-03-roadmap %}) post outlining development opportunities.
+
+First of all thanks to developers and organisations that have responded with offers of in-kind contributions. We have not had enough capacity to organize in-kind contributions or fundraising and this blog post is an attempt to remedy.
+
+This post highlights development activities that are **available to be worked on today**, along with interested [developers](https://github.com/geoserver/geoserver/graphs/contributors) and commercial [support providers](/support) available to work on GeoServer roadmap items.
+
+## Spring Framework 6 Tasks
+
+The key challenge we are building towards is a spring-framework 6 update, ideally by the end of 2024 when the version we use now reaches end-of-life.
+
+The tasks below are steps towards this goal.
+
+### Wicket 9 upgrade
+
+Interested Parties:
+
+* Brad has been doing amazing work with the Wicket 9 upgrade and is in need of assistance.
+* GeoCat has offered to do manual A/B testing when PR is ready for testing
+
+Activity:
+
+* [[GEOS-11275]](https://osgeo-org.atlassian.net/browse/GEOS-11275) Wicket 9 upgrade
+* [geoserver#7154](https://github.com/geoserver/geoserver/pull/7154)
+
+### Spring Security 5.8 update
+
+Spring Security 5.8 provides a safe stepping stone ahead of the complete spring-framework 6 upgrade and is an activity that can be worked on immediately. 
+
+Interested parties:
+- Andreas (ITS Digital Solutions GmbH) offered to work on this activity in during the initial January call out, and has indicated they are now ready to start.
+- GeoSolutions has indicated they are in position to work on this activity if financing is available
```

Comment:
@aaime you did not provide anything specific GeoSolutions was in position to work on (if financing is available). Is this good enough or would you like to add to more of the items below.

Reply:
Given someone is doing this work already, I'd remove the GeoSolutions line, it's unnecessary, best to use the available manpower on other tasks instead.